src/ec/packages/ec-component-site-header-standardised/__snapshots__/site-header-standardised.test.js.snap

```diff
@@ -13,17 +14,9 @@ exports[`EC - Site Header Standardised Default renders correctly 1`] = `
         class="ecl-site-header-standardised__top"
       >
         <a
-          aria-label="European Commission"
           class="ecl-link ecl-link--standalone ecl-site-header-standardised__logo-link"
-          href="/example"
-        >
-          <img
-            alt="European Commission logo"
-            class="ecl-site-header-standardised__logo-image"
-            src="/logo--en.svg"
-            title="European Commission"
-          />
-        </a>
+          href=""
+        />
```

Comment:
no more link, aria-label here seems strange.

Reply:
yeah, it was wrong the template, only of the standardised, fixed it ;)